spec/helpers/posts_helper_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
```

Comment:
I would go ahead and remove this. They often don't get used and you can alwaus add it back in if you end up writing helper methods.